camp_1/README.md

```diff
@@ -904,39 +923,247 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
+3. Invoke the function `deployContract` of the UDC on either Goerli or Mainnet, passing the class hash of your contract and the constructor arguments, among other parameters.
+   
+Notice that a "deploy" transaction is never used; instead, the invoke transaction is used. The invoke transaction requires a user account (wallet) with enough wrapped Ether to charge the user for the deployment.
+
+Before deploying our contract, we need to define the account contract that will pay for the deployment. Using the `starknet` CLI, we can define a new wallet with `new_account`. Using the '--account' flag, we can name the wallet with an alias. We can use the alias to refer to the account in later commands; in the example below, we call it `voting-contract`. With the `--wallet` flag, we can indicate the wallet format we will use for our account contract. Since we are using Account Abstraction, account contracts are programable (not just a fixed EOA as with the L1). That is, we can give customized functionality to our account contract. In this case, we are using the standard Open Zeppelin (modified by Starkware) account contract: `starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount`. Before, when we were using the Argent UI, we implicitly decided that we would use the Argent's account contract with its functionality and features, which differs from Open Zeppelin's. We will learn more about account contracts and how to customize them in Camp 3. This is one of the most powerful features in StarkNet and is the catalyst to get a Web2-level User Experience (UX).
+
+```Bash
+starknet new_account --network alpha-goerli --account voting-contract --wallet starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount
+```
+
+The `new_account` command did not deploy the account contract; it simply calculated its address and public key:
+
+```Bash
+Account address: 0x00f20c6664cc47e569abe53c7ba19f04685158a1b2f01c9a923cd3849354a928
+Public key: 0x01dadfed69928d423a4328ca1a4cd9b375a5911bb810fd2af9f5cd4ec914aec5
+Move the appropriate amount of funds to the account and then deploy the account
+by invoking the 'starknet deploy_account' command.
+
+Note: This is a modified version of the OpenZeppelin account contract. The signature is computed
+differently.
+```
+
+Before deploying this account contract, we need to fund it so it can pay for its deployment (this process is called counterfactual deployment and will be studied in Camp 3). We can go back to the [StarkNet Goerli Faucet](https://faucet.goerli.starknet.io/) and feed it with the address we got from `new_account`: `0x00f20c6664cc47e569abe53c7ba19f04685158a1b2f01c9a923cd3849354a928`. We wait a couple of minutes, and then we can deploy our account contract:
+
+```Bash
+starknet deploy_account --network alpha-goerli --account voting-contract --wallet starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount
```

Comment:
have you tried using `\` to separate a long command like this one into multiple lines for better readability?

Reply:
Agree! I added this to all CLI commands! It is much better.